### PR TITLE
[RF] POM > AT_09.03.001 | 'Edit description button is visible'

### PR DESCRIPTION
**Pull request by asyniagina**
https://trello.com/c/p6dZyaI4/1988-rf-pom-at0903001-my-viewsedit-description-link-is-visible

### CODE REVIEW

**Comment by MarinaMone** [comment]
@asyniagina It has to be two different tests . One for clickable , one for visible. Two  can not be in one test

**Comment by asyniagina** [comment]
> @asyniagina It has to be two different tests . One for clickable , one for visible. Two can not be in one test

@MarinaMone I fixed it, thank you

File: cypress/e2e/tests/myView.cy.js [hunk 1]
```diff
@@ -219,6 +219,17 @@ describe('myView', () => {
...
   });
 
+  it('AT_09.03.001 | Edit description link is visible', () => {
+    homePage
+        .clickMyViewSideMenuLink()
+        .clickAddDescriptionBtn()
+        .typeDescriptionIntoInputField(myViewData.addDescription)
+        .getEditDescriptionLink()
+        .should('be.visible')
+        .and('contain', 'Edit description')
```
Comment (Rom-T) [inline cypress/e2e/tests/myView.cy.js:229]: Please move `Edit description` in the fixtures

File: cypress/e2e/tests/myView.cy.js [hunk 2]
```diff
@@ -219,6 +219,17 @@ describe('myView', () => {
         .should('contain', myViewData.addDescription)
   });
 
+  it('AT_09.03.001 | Edit description link is visible', () => {
```
[the remark this answers is not in the document]
Comment (IrynaKolh) [reply cypress/e2e/tests/myView.cy.js:222]: .getEditDescriptionLink() дублируется на 30 и 38 строку кода в файле MyViewPage.js, может стоит один удалить? или переименоват 1 из нах в .getEditDescriptionBtn()

File: cypress/e2e/tests/myView.cy.js (hunk 1 above)
Comment (asyniagina) [reply cypress/e2e/tests/myView.cy.js]: > Please move `Edit description` in the fixtures

@Rom-T thank you, I fixed it

File: cypress/e2e/tests/myView.cy.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (asyniagina) [reply cypress/e2e/tests/myView.cy.js]: > I think, Edit description is a button, not a link

@IrynaKolh thank you, I fixed it, and second note

**Review by AlexSQE — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Review by IrynaKolh — APPROVED** [review]

**Pull request merged**